client/components/home/Homepage.js

```diff
@@ -85,15 +93,18 @@ class Homepage extends React.Component {
     let paginationInfo;
     let pageCount;
     let currentPage;
+    const userDocumentsInStore = this.props
+      .currentState.displayUserDocuments;
+    const searchStatus = this.props.currentState.searchParams;
 
-    if (this.props.userDocumentsInStore.documents !== undefined) {
-      if (this.props.userDocumentsInStore.paginationInfo !== undefined) {
-        paginationInfo = this.props.userDocumentsInStore.paginationInfo;
+    if (userDocumentsInStore.documents !== undefined) {
+      if (userDocumentsInStore.paginationInfo !== undefined) {
```

Comment:
I'm guessing this is coming from your store, which gets data from your backend. If I'm right why would paginationInfo be undefined? should it not be attached to whatever the api returns?